centos-7.template

```diff
@@ -117,9 +118,10 @@ cat > cert-gen.sh.base64 << EOF
 ${cert_gen}
 EOF
 
-base64 -d < cert-gen.sh.base64 > cert-gen.sh
-chmod +x cert-gen.sh
-mv cert-gen.sh /opt
+base64 -d < cert-gen.sh.base64 > minishift-cert-gen
+rm cert-gen.sh.base64
```

Comment:
the action is performed as root on a regular file. if it fails, something bad is happening...